app/ai/voice/llm/realtime/azure_realtime.py

```
@@ -3,27 +3,153 @@
-    AudioOutput,
-    InputAudioNoiseReduction,
-    InputAudioTranscription,
-    SemanticTurnDetection,
+    ResponseCreateEvent,
     SessionProperties,
 )
 
+try:
+    from websockets.asyncio.client import connect as websocket_connect
+except ModuleNotFoundError as e:
+    raise Exception(f"Missing module: {e}")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)